Syntaxes/debian-control.sublime-syntax

```diff
@@ -1,53 +1,63 @@
-    )
-
-  # https://www.debian.org/doc/manuals/debmake-doc/ch05.en.html#substvar
-  allowed_substvar: |-
-    (
-      |\${misc:Depends}|\${misc:Pre-Depends}|\${shlibs:Depends}|\${python:Depends}|
-      |\${python3:Depends}|\${perl:Depends}|\${ruby:Depends}|
-    )
-
 contexts:
   main:
-    - include: comments
-    - include: email-address
+    - include: debian-common.sublime-syntax#comments
     - include: fields
-    - include: substvar
 
-  comments:
-    - match: "#"
-      scope: punctuation.definition.comment.debian
+  # https://www.debian.org/doc/debian-policy/ch-controlfields.html#source-package-control-files-debian-control
+  fields:
+    - match: |-
+        ^(?xi:
+          (Source|Maintainer|Uploaders|Section|Priority|Build-Depends(?:-Arch|-Indep)?
+            |Build-Conflicts(?:-Arch|-Indep)?|Standards-Version|Homepage
+            |Vcs-(?:Browser|Arch|Bzr|Cvs|Darcs|Git|Hg|Mtn|Svn)|Testsuite|Rules-Requires-Root
+            |Package|Architecture|Essential|Depends|Pre-Depends|Recommends|Suggests|Breaks
+            |Conflicts|Provides|Replaces|Enhances|Description|Homepage|Built-Using|Package-Type
+        ))(:)?
+      captures:
+        1: meta.mapping.key.debian keyword.control.debian.control
+        2: punctuation.separator.mapping.key-value.debian.control
       push:
-        - meta_scope: comment.line.debian
-        - match: \n
+        - meta_content_scope: meta.mapping.value.debian.control
+        - match: ^(?=\S)
           pop: true
+        - include: substvar
+        - include: semver-flag
+        - include: debian-common.sublime-syntax#email-address
+        - include: debian-common.sublime-syntax#url
+        - match: \,|\|
+          scope: punctuation.separator.sequence.debian.control
 
-  email-address:
-    - match: <.+@.+[.].+>|<.+?at.+?dot.+?>
-      scope: variable.language.debian
-
-  fields:
-    - match: '{{allowed_fields}}'
-      scope: keyword.control.debian
-
+  # https://www.debian.org/doc/manuals/debmake-doc/ch05.en.html#substvar
   substvar:
-    - match: '{{allowed_substvar}}'
-      scope: variable.function.debian
+    - match: |-
+        (?x:
+          (\${)
+          (?:misc:Depends|misc:Pre-Depends|shlibs:Depends|python:Depends
+            |python3:Depends|perl:Depends|ruby:Depends)
+          (})
+        )
+      scope: variable.language.debian
+      captures:
+        1: punctuation.definition.variable.begin.debian.control
+        2: punctuation.definition.variable.end.debian.control
 
+  semver-flag:
+    - match: (?=\(>?=[ ]+\d(?:\.\d){0,2})
+      push:
+        - match: (\()(>?=)\s+
+          captures:
+            1: punctuation.section.group.begin.debian.control
+            2: keyword.operator.arithmetic.debian.control
+        - include: debian-common.sublime-syntax#semantic-version
+        - match: (\))
+          scope: punctuation.section.group.end.debian.control
+          pop: true
+        - include: debian-common.sublime-syntax#pop-eol
```

Comment:
Could (should?) this be extended to match Debian's [Standards-Version](https://www.debian.org/doc/debian-policy/ch-scope.html#new-versions-of-this-document)

Reply:
You mean [this](https://www.debian.org/doc/debian-policy/ch-controlfields.html#standards-version), right?

Reply:
Yes Sir. The link I referred to shows the actual package and it's version numbers.